### PR TITLE
Implementation of jnp.squeeze

### LINKED ISSUE #7427: squeeze

### PR DESCRIPTION
**Pull request by karalleyna**
Closes #7427

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
